docs/manual/adapting-existing.md

```diff
@@ -71,17 +71,39 @@ directory.
 
 ## Booting and initramfs technology
 
-OSTree comes with optional dracut+systemd integration code that parses
-the `ostree=` kernel command line argument in the initramfs, and then
-sets up the read-only bind mount on `/usr`, a bind mount on the
-deployment's `/sysroot` to the physical `/`, and then finally uses
-`mount(MS_MOVE)` to make the deployment root appear to be the root
-filesystem before telling systemd to switch root.
+OSTree comes with optional dracut+systemd integration code:
```

Comment:
I think you need a space between this line and the one below, otherwise it's not going to render as a list.